### DISCUSSION TITLE
CompositeCollection behavior with DynamicData

### DISCUSSION THREAD

**Discussion by Tenjim** [reactivemarbles/DynamicData#542] [Q&A]
Hi guys,

Is it possible to doing something like CompositeCollection from wpf with DynamicData ?
I wonder if we can join all collections with DynamicData with different types.
my goal is to bind few collections into ItemsControl.Items. 

Currently, I'm using Avalonia with ReactiveUI.

**Comment by RolandPheasant** [comment]
There are several  operators in DD which can be used combine multiple sources.

These combine multiple operators: Or, And and Except. These apply a logical combinations of multiple sources into a single source
